views/lib/ldap-info.hbs

```diff
@@ -1,6 +1,5 @@
 <div class="col-sm-12 alert alert-info">
-	<strong>Warum kann ich keine Einträge bearbeiten?</strong><br>
-	Deine Schule in der {{theme.short_title}} ist mit einem Identitätsmanagement (LDAP, Landessystem o.ä.) synchronisiert. 
-	Die Daten von Lehrkräften und Schülern werden dort zentral verwaltet und können nur dort geändert werden. 
-	Das schließt auch Passwortänderungen mit ein.
+	<strong>{{$t "lib.idap_info.text.whyCantIEditEntries" }}</strong><br>
+	{{$t "lib.idap_info.text.yourSchoolInThe" }} {{theme.short_title}} {{$t "lib.idap_info.text.isSyncronizedWithAnIdentity" }} 
```

Comment:
as described on [https://docs.schul-cloud.org/pages/viewpage.action?pageId=113643606](url) merge into:
{{$t "lib.ldap_info.text.yourSchoolIsLinkedToExternManagementSystem" (dict "shortTitle" theme.short_title) }}

in Json:
"yourSchoolIsLinkedToExternManagementSystem" : "Deine Schule in der {{shortTitle}} ist mit einem Identitätsmanagement (LDAP, Landessystem o.ä.) synchronisiert. "